app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -392,15 +392,13 @@ class BloodPressureEntrySheet : BottomSheetActivity(), BloodPressureEntryUi, Rem
 
```

Comment:
Can you rename the commit message to "Move `ProgressBar` inside the parent layout" please?